temporalcli/commandsmd/commands.md

```diff
@@ -1,487 +1,1185 @@
+* What's the best way to do async conversations other than PRs?
+
+NEW: 
+* `--set-as-default` (bool) -
+This is super confusing. When you add the new Build ID, it becomes the default
+for the set it is inserted into. This command makes the set become the Task
+Queue default, and the option defaults to false. Suggestion: --promote-set-as-default-set
+or something like that because mind blown.
+
+* Word wrapping to 80 chars
+    * What about options? they tend to run long
+    * Could the text go to a second line after the spaced-dash?
+    * Could long descriptions be autowrapped outside code examples? -- NEW Q
+* --help, --long-help, --full-help to provide progressive support?
+* --long, --extra-long/--full and not --really-long
+  * `--long`, `-l` (bool) -
+  * `--really-long` (bool) - 
+* Auditing confirmation elements, especially bools
+  * `--yes`, `-y` (bool) - has multiple meanings based on use
+    (default true, false varies)
+* Payloads
+  * `--no-json-shorthand-payloads` (bool) - Shorter? --literal-payload? --string-payload?
+  * `--input-base64` (bool) - --data-base64?
+
+* These all seem to do with presentation
+  * `--detail` (bool) -
+  * `--event-details` (bool) -
+  * `--log-config` (bool) -
+  * `--no-fold` (bool) -
+  * `--raw` (bool) -
+  * `--follow`, `-f` (bool) -
+  * `--reset-points` (bool) - ? --show-reset-points?
+
+(NEW) Inconsistent use of identical items
+
+* `--build-id` (string) -
+* `--build-id` (string[]) -
+* `--cron` (string) -
+* `--cron` (string[]) -
+* `--data` (string) -
+* `--data` (string[]) -
+* `--detail` (bool) -
+* `--detail` (string) -
+* `--name` (string) -
+* `--name` (string[]) -
+* `--namespace`, `-n` (string) -
+* `--namespace`, `-n` (string[]) -
+* `--task-queue`, `-t` (string) -
+* `--task-queue`, `-t` (string[]) -
+* `--tls-ca-data` (string) -
+* `--tls-ca-path` (string) -
+* `--tls-cert-data` (string) -
+* `--tls-cert-path` (string) -
+* `--tls-key-data` (string) -
+* `--tls-key-path` (string) -
+* `--type` (string) -
+* `--type` (string[]) -
+* `--type`, `-t` (string-enum) -
+
+
+* Consistency?
+  * `--pause` (bool) -
+  * `--unpause` (bool) -
+  * `--pause-on-failure` (bool) -
+  * `--paused` (bool) - Perhaps: set-paused? set-paused-state?
+
+* `--global` (bool) -
+* `--promote-global` (bool) -
+
+* `--enable-connection` (bool) - ? --connected? --set-connected?
+* `--fail-existing` (bool) - ? --fail-current?
+
+* `--archived` (bool) - ? --set-archived?
+
+* `--headless` (bool) -
+
+* `--set-as-default` (bool) -
+ 
+* `--tls-disable-host-verification` (bool) -
+* `--tls` (bool) -
+
+----
+
+Limiting returns
+* `--limit` (int) -
+* `--max-sets` (int) - ?? Could be limit also
+* `--depth` (int) -
+
+* `--concurrency` (int) -
+* `--event-id`, `-e` (int) -
+* `--partitions` (int) -
+* `--remaining-actions` (int) -
+
+Ports (looks good)
+* `--http-port` (int) -
+* `--metrics-port` (int) -
+* `--port`, `-p` (int) -
+* `--ui-port` (int) -
+
+
+----
```

Comment:
We can start a separate (internal) Notion page with all of the above, I think; some/many of these are breaking changes, so we need to decide if it's worth breaking people's scripts and such for this. But we should definitely capture all of this in one place (along with what you had in your Google doc). LMK if you want me to start a page somewhere.

Reply:
I'm going to use the CLI feedback Google doc to start with as an info dump first. It's just me collecting thoughts right now and I do want to move that out of the source file and I don't want to spend any time/overhead with formatting and clarifying yet. That can wait until I finish my first pass through (prelude to sanity checks, reviews, etc)